src/qt/overviewpage.cpp

```diff
@@ -61,11 +61,6 @@
 #define DECORATION_SIZE 64
 #define NUM_ITEMS 6 // 3 - number of recent transactions to display
 
-  extern uint64_t global_balance_money_maineco[100000];
```

Comment:
No need to `extern` the variables here, because they are already already provided via `mastercore.h`.